src/OpenTelemetry.Extensions.Docker/DockerExtensionsEventSource.cs

```diff
@@ -0,0 +1,63 @@
+// <copyright file="DockerExtensionsEventSource.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Threading;
+
+namespace OpenTelemetry.Extensions.Docker
+{
+    [EventSource(Name = "OpenTelemetry-Extensions-Docker")]
+    internal class DockerExtensionsEventSource : EventSource
+    {
+        public static DockerExtensionsEventSource Log = new DockerExtensionsEventSource();
+
+        [NonEvent]
+        public void ExtractResourceAttributesException(string format, Exception ex)
+        {
+            if (this.IsEnabled(EventLevel.Error, (EventKeywords)(-1)))
+            {
+                this.FailedToExtractResourceAttributes(format, ToInvariantString(ex));
+            }
+        }
+
+        [Event(1, Message = "Failed to extract resource attributes in '{0}'.", Level = EventLevel.Error)]
```

Comment:
only 1 parameter?

Reply:
It Prints out the LOG Level with filename and an actual exception. Do you expect more?